<launch>

  	<arg name="camera" default="/stereo_down"/>
	<arg name="decimation" default="2" />
	<arg name="node_name" default="inference_yolov5" />
	<arg name="robot_name" default="stereo_ch3" />

	 <!-- load parameters -->
	<rosparam command="load" file="/home/plomenuc/plome_ws/src/stereo_plome/config/model_r0.yaml"/>
	<!-- detect image -->
	<node name= "$(arg node_name)" pkg="stereo_plome" type="yolov8_IS_inference.py" output="screen">
	<!-- <node name= "$(arg node_name)" pkg="stereo_plome" type="plome_yolov5_inference.py" output="screen"> -->
		<param name="robot_name" value="$(arg robot_name)"/>
		<!-- <remap from="image_rect_color" to="/$(arg robot_name)/left/image_raw"/> -->
		<remap from="image_rect_color" to="/$(arg robot_name)/left/image_color"/>
        <remap from="camera_info" to="/$(arg robot_name)/left/camera_info"/>
        <!-- <remap from="camera_info" to="/$(arg robot_name)/left/camera_info"/> -->
		<remap from="navigator/navigation" to="/$(arg robot_name)/navigator/navigation"/>
	</node>
</launch>


